src/process.cr

```diff
@@ -31,19 +47,26 @@ class Process
     nil
   end
 
-  def self.ppid : LibC::PidT
-    LibC.getppid
+  # Returns true if the process identifier given by *pid* is valid for
+  # a currently running process, false otherwise.
+  def self.exists?(pid : Int)
```

Comment:
From the linux man page (conforming to POSIX), if the process or process group does not exist then errno is set to `ESRCH`, so shouldn't this be:

```crystal
def self.exists?(pid : Int)
  if LibC.kill(pid, 0) == 0
    return true
  end
  if Errno.value == Errno::ESRCH
    return false
  end
  raise Errno.new("kill")
end
```